### PR TITLE
Upgrade 1.3.0

### PR DESCRIPTION
**Pull request by girazoki**
Upgrades dependencies to 1.3.0

### CODE REVIEW

File: runtime/dancebox/src/lib.rs [hunk 1]
```diff
@@ -386,15 +386,6 @@ parameter_types! {
     pub const ExistentialDeposit: Balance = EXISTENTIAL_DEPOSIT;
 }
 
-/// A reason for placing a hold on funds.
```
Comment (girazoki) [inline runtime/dancebox/src/lib.rs:389]: TODO: check in Fork tests this does not create issues with existing holds!

File: container-chains/templates/frontier/runtime/src/lib.rs [hunk 2]
```diff
@@ -788,6 +789,7 @@ parameter_types! {
     pub BlockGasLimit: U256 = U256::from(BLOCK_GAS_LIMIT);
     pub PrecompilesValue: TemplatePrecompiles<Runtime> = TemplatePrecompiles::<_>::new();
     pub WeightPerGas: Weight = Weight::from_parts(weight_per_gas(BLOCK_GAS_LIMIT, NORMAL_DISPATCH_RATIO, WEIGHT_MILLISECS_PER_BLOCK), 0);
+    pub SuicideQuickClearLimit: u32 = 0;
```
Comment (girazoki) [inline container-chains/templates/frontier/runtime/src/lib.rs:792]: check what this implies!

Comment (girazoki) [reply container-chains/templates/frontier/runtime/src/lib.rs:792]: it is to delete the storage associated with the contracts up to the limit: https://github.com/paritytech/frontier/pull/1221.

Setting it to 0 maintains the same behavior as before, but for future releases, maybe we should think about setting this value

File: runtime/dancebox/src/lib.rs (hunk 1 above)
Comment (girazoki) [reply runtime/dancebox/src/lib.rs:389]: We need a migration for this, running the fork tests has made me realize that the enums are not mapped equally!

Comment (girazoki) [reply runtime/dancebox/src/lib.rs:389]: Migration added already

**Review by tmpolaczyk — APPROVED** [review]

File: runtime/dancebox/tests/common/xcm/foreign_signed_based_sovereign.rs [hunk 3]
```diff
@@ -50,6 +50,10 @@ fn using_signed_based_sovereign_works_in_tanssi() {
     };
 
     let xcm = VersionedXcm::from(Xcm(vec![
+        DescendOrigin(X1(AccountId32 {
```
Comment (girazoki) [inline runtime/dancebox/tests/common/xcm/foreign_signed_based_sovereign.rs:53]: This sucks but westend no longer lets you send a message form a signed origin

File: runtime/dancebox/src/migrations.rs [hunk 4]
```diff
@@ -375,6 +375,125 @@ where
...
+                    <T as pallet_balances::Config>::RuntimeHoldReason,
+                    <T as pallet_balances::Config>::Balance,
+                > = pallet_balances::IdAmount {
+                    id: pallet_pooled_staking::HoldReason::PooledStake.into(),
+                    amount: hold.amount,
+                };
+                new_holds.push(new_item);
+            }
+            let bounded = BoundedVec::<_, T::MaxHolds>::truncate_from(new_holds.clone());
+            pallet_balances::Holds::<T>::insert(&account_id, bounded);
```
Comment (fgamundi) [inline runtime/dancebox/src/migrations.rs:436]: Should the old data be cleared from storage?

Comment (girazoki) [reply runtime/dancebox/src/migrations.rs:436]: we are writing into the same storage, but we are just overriding it with the new type

**Review by fgamundi — APPROVED** [review]

**Pull request merged**